MGDOBR-47: add demo and filters documentation (#49)
5733d9e2be6485d52ffa08870cabdee0/sandbox: DEMO.md

> # DEMO

Reviewer: Ok it is Friday afternoon and it is also beer o'clock but I was thinking to convert this file (and filters too) as a jupyther notebook and execute it as part of CI job during PR checks to making sure it is still working 😃 
It is mainly a set of curl/REST calls to be converted as Python code :)
Wdyt? 
![image](https://user-images.githubusercontent.com/244607/134683867-afb1d16b-cf78-4d6d-82b8-806ba058ddd7.png)

Author: Makes sense, I've created https://issues.redhat.com/browse/MGDOBR-68

Check the status of the deployment with a GET request to the `/api/v1/bridges/{id}` endpoint: 

```bash
curl -X GET $MANAGER_URL/api/v1/bridges/$BRIDGE_ID
```

the response should look like 

```json
{
  "kind":"Bridge",
  "id":"87508471-ee0f-4f53-b574-da8a61285986",
  "name":"myBridge",
  "href":"/api/v1/bridges/87508471-ee0f-4f53-b574-da8a61285986",
  "submitted_at":"2021-09-24T11:29:33.086649+0000",
  "status":"AVAILABLE",
  "endpoint":"/ingress/events/87508471-ee0f-4f53-b574-da8a61285986"
}
```

The application is now `AVAILABLE` and we also have the information about the endpoint to use to push the events: `/ingress/events/87508471-ee0f-4f53-b574-da8a61285986` in this particular case.

## How to add Processors to the Bridge

It is possible to add Processors to the Bridge instance using the endpoint `/api/v1/bridges/{bridgeId}/processors`. 
There are 3 basic concepts around a Processor:

- **filters**: when one or more filters are specified, only those events that evaluates the filters to `true` will be processed by the specified action. The documentation around the filters can be found [here](FILTERS.md).

> - **transformations**: if a transformation schema has been specified in the request, the original event is transformed according to the schema before it is sent to the action.
> - **actions**: the action to take with the filtered events.

Reviewer: I would prefer to switch the order: transformation concept is tied to action. Wdyt?
```suggestion
- **actions**: the action to take with the filtered events.
- **transformations**: an optional transformation logic can be specified for each action definition.
```
